### DISCUSSION TITLE
增加字幕的批量翻译

### DISCUSSION THREAD

**Discussion by cmulittlechild** [immersive-translate/immersive-translate#1301] [Ideas]
### 新功能 | New Feature

字幕翻译能否增加批量导入翻译导出的功能？

### 为什么您想要添加这些功能 | Why you want to add these features

经常有很多字幕需要翻译，一个一个添加导出太麻烦了。

**Comment by theowenyoung** [comment]
请问下一般是什么场景 会有批量的需求？

**Reply by cmulittlechild** [reply]
比如我最近因为英语学习的需求，会将十几集甚至几百集的视频一次性转录出英文字幕，此时如果一个一个导入导出就太繁琐了，因此问一下是否可以增加批量的功能。

**Reply by theowenyoung** [reply]
可以问一下你是用什么转录工具的嘛？转录的时候工具提供了批量转录是吗？

**Reply by cmulittlechild** [reply]
我使用的是mac端的Yplayer，本地引擎，软件提供了批量转录功能，可以一次性将文件夹扔进去，转录完字幕自动导出到当前文件夹。
